package.json

```diff
@@ -11,7 +11,8 @@
     "test": "turbo run test --no-cache",
     "dev": "turbo run dev --parallel",
     "lint": "turbo run lint --parallel",
-    "lint:fix": "turbo run lint:fix --parallel"
+    "lint:fix": "turbo run lint:fix --parallel",
+    "changeset": "changeset"
```

Comment:
wondering if we need that as far as i know then your run `pnpm <something>` it also looks into the `node_modules/bin` folder.
https://pnpm.io/using-changesets

Reply:
you are right! I will remove this one, but leave it in the Readme